repos/fdbt-site/src/components/PurchaseMethodCard.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { ReactElement } from 'react';
+import { PurchaseMethodCardBody } from '../pages/viewPurchaseMethods';
+import { SalesOfferPackage } from '../interfaces/matchingJsonTypes';
+
+interface PurchaseMethodCardProps {
+    sop: SalesOfferPackage;
+    isCapped: boolean;
+    index: number;
+    deleteActionHandler?: (id: number, name: string) => void;
+}
+
+const PurchaseMethodCard = ({ sop, isCapped, index, deleteActionHandler }: PurchaseMethodCardProps): ReactElement => {
+    const { name, id } = sop;
+
+    return (
+        <div className="card" key={id} id={`purchase-method-${isCapped ? 'cap' : ''}-${index}`}>
+            <div className="card__body">
+                {deleteActionHandler ? (
+                    <div className="card__actions">
+                        <ul className="actions__list">
+                            <li className="actions__item">
+                                <a
+                                    className="govuk-link govuk-!-font-size-16 govuk-!-font-weight-regular"
+                                    href={`/managePurchaseMethod?id=${id}${isCapped ? `&capped=true` : ''}`}
```

Comment:
The query string must either be capped=true or isCapped=true in ALL places, we can't have some being different to others.